README.md

```diff
@@ -4,18 +4,24 @@ Simple parallel test runner for Julia tests with autodiscovery.
 
 ## Usage
 
-```julia
-julia> import Pkg
+The main entry point of `ParallelTestRunner` is the exported function `runtests`, which takes two mandatory arguments:
+
+* the module for which you want to run the tests
+* the list of arguments passed to the test script, as a `Vector{String}`, this will typically be [`Base.ARGS`](https://docs.julialang.org/en/v1/base/constants/#Base.ARGS).
 
-julia> Pkg.test("ParallelTestRunner"; test_args=["--help"])
+With the `--help` flag you can print a help message:
+
+```julia
+julia> using ParallelTestRunner
 
+julia> ParallelTestRunner.runtests(ParallelTestRunner, ["--help"])
```

Comment:
Note that `Pkg.test("ParallelTestRunner"; test_args=["--help"])` actually runs `ParallelTestRunner` tests without printing any help message, so that was quite misleading 🙂